docs/prow/prow-architecture.md

```diff
@@ -9,11 +9,12 @@ See an overview of the Prow production cluster, its components, and interactions
 ![Prow architecture overview](./assets/prow-architecture.svg)
 
 ## Provisioning and Secrets
-The [set-up-workload-cluster.sh](../../prow/set-up-workload-cluster.sh) script provisions the cluster where Prow runs its workload Pods. To enable the Prow cluster to schedule jobs in the workload cluster, you must create a Secret using [create-secrets-for-workload-cluster.sh](../../prow/create-secrets-for-workload-cluster.sh) script. The Prow main cluster gets provisioned by the [install-prow.sh](../../prow/install-prow.sh) script that the Prow administrator executes manually. The script reads the full configuration from the `test-infra` repository. Based on that, the administrator can recreate the clusters at any time. That is why, when you make any changes to the cluster configuration, apply them directly on the repository instead of the runtime. Only the Prow administration team needs access to the runtime. All others should contribute changes through the repository with the review process in place. The administrators upload any new configuration to the cluster.
+The [set-up-workload-cluster.sh](../../prow/set-up-workload-cluster.sh) script provisions the cluster where Prow runs its workload Pods. To enable the Prow cluster to schedule jobs in the workload cluster, you must create a Secret using the [create-secrets-for-workload-cluster.sh](../../prow/create-secrets-for-workload-cluster.sh) script. 
+The Prow administrator manually executes the [install-prow.sh](../../prow/install-prow.sh) script to provision the Prow main cluster. The script reads the full configuration from the `test-infra` repository. Based on that, the administrator can recreate the clusters at any time. If you want to introduce changes to the configuration, do not change anything directly on the cluster. Commit your changes to the `test-infra` repository so that a reviewer can check them. The administrators upload any new configuration to the cluster.
```

Comment:
```suggestion
The Prow administrator manually executes the [install-prow.sh](../../prow/install-prow.sh) script to provision the main Prow cluster. The script reads the full configuration from the `test-infra` repository. Based on that, the administrator can recreate clusters at any time. If you want to introduce changes to the configuration, do not change anything directly on the cluster. Commit your changes to the `test-infra` repository so that a reviewer can check them. The administrators upload any new configuration to the cluster.
```

Reply:
Done